tests/forms_tests/widget_tests/test_radioselect.py

```diff
@@ -6,9 +6,15 @@
 class RadioSelectTest(WidgetTest):
     widget = RadioSelect
 
+    def test_format_value(self):
```

Comment:
I think these tests don't belong in this patch considering they already pass and `ChoiceWidget.format_value()` isn't modified.